Dockerfile

```diff
@@ -1,5 +1,7 @@
-FROM golang:1.10.3-stretch AS build
-WORKDIR src/github.com/mesg-foundation/core
+FROM golang:1.11.4-stretch AS build
+WORKDIR /core
+COPY go.mod go.sum ./
```

Comment:
Why copy twice? It will be copied with the `COPY . .`

Reply:
as said in discord, it's for using creating a docker cache only for dependencies